tests/functional/test_install.py

```diff
@@ -531,24 +531,23 @@ def test_install_package_which_contains_dev_in_name(script):
     assert egg_info_folder in result.files_created, str(result)
 
 
-@pytest.mark.network
 def test_install_package_with_target(script):
     """
     Test installing a package using pip install --target
     """
     target_dir = script.scratch_path / 'target'
-    result = script.pip('install', '-t', target_dir, "initools==0.1")
+    result = script.pip_install_local('-t', target_dir, "initools==0.1")
```

Comment:
why add another initools?   there's simple v1/v2/v3 which should work here and for the upgrade below?